### PR TITLE
feat: Make clusters configurable via settings

### PR DESCRIPTION
**Pull request by lynnagara**
Clusters are now configured via settings. The CLICKHOUSE_SERVER,
CLICKHOUSE_HOST, CLICKHOUSE_PORT and CLICKHOUSE_HTTP_PORT settings
are now deprecated in favor of cluster specific values provided
via the CLUSTERS list.

We will also need to add the CLUSTERS configuration for Sentry production and Single tenant.

### CODE REVIEW

**Comment by fpacifici** [comment]
No issue with this, will accept as soon as the production and single tenant changes are done.

**Comment by lynnagara** [comment]
This has been updated in prod. And single tenant can be done as part of the next Snuba release so it's not a blocker.

File: snuba/settings.py [hunk 1]
```diff
@@ -15,15 +15,17 @@
 DATASET_MODE = "local"
 
 # Clickhouse Options
-# TODO: Warn about using `CLICKHOUSE_SERVER`, users should use the new settings instead.
-[default_clickhouse_host, default_clickhouse_port] = os.environ.get(
-    "CLICKHOUSE_SERVER", "localhost:9000"
-).split(":", 1)
```
Comment (fpacifici) [inline snuba/settings.py:21]: I assume you verified the CLICKHOUSE_SERVER parameter is not used anywhere.

**Review by fpacifici — APPROVED** [review]

File: snuba/settings.py (hunk 1 above)
Comment (lynnagara) [reply snuba/settings.py:21]: Yeah. Checked again just to be sure.

**Pull request merged**